### ISSUE TITLE
Unable to start telepresence 0.67 on arch linux

### ISSUE THREAD

**Issue by vstepchik** [datawire/telepresence#302]
### What were you trying to do?

Start telepresence to access services deployed on local minikube.
Telepresence is installed from AUR package: https://aur.archlinux.org/packages/telepresence/
Not sure what additional info could help, please ask if you need something.

### What did you expect to happen?

Telepresence started

### What happened instead?

It crashed

### Automatically included information

Command line: `['/usr/bin/telepresence', '--logfile', '/tmp/telepresence.log']`
Version: `0.67`
Python version: `3.6.2 (default, Jul 20 2017, 03:52:27) 
[GCC 7.1.1 20170630]`
kubectl version: `Client Version: v1.7.6`
oc version: `(error: [Errno 2] No such file or directory: 'oc')`
OS: `Linux vstepchik_macpro 4.12.13-1-ARCH #1 SMP PREEMPT Fri Sep 15 06:36:43 UTC 2017 x86_64 GNU/Linux`
Traceback:

```
Traceback (most recent call last):
  File "/usr/bin/telepresence", line 257, in call_f
    return f(*args, **kwargs)
  File "/usr/bin/telepresence", line 2350, in go
    runner, args
  File "/usr/bin/telepresence", line 1516, in start_proxy
    processes, socks_port, ssh = connect(runner, remote_info, args)
  File "/usr/bin/telepresence", line 1099, in connect
    bufsize=0,
  File "/usr/bin/telepresence", line 404, in popen
    return self.launch_command(track, *args, **kwargs)
  File "/usr/bin/telepresence", line 367, in launch_command
    stderr=self.logfile
  File "/usr/lib/python3.6/subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1333, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'stamp-telepresence'

```

Logs:

```
kube', '--namespace', 'default', 'get', 'pod', 'telepresence-1506520541-048964-25778-1327857849-08rns', '-o', 'json'],)...
  18.4 TL | [58] captured.
  18.6 TL | [59] Capturing: (['kubectl', '--context', 'minikube', '--namespace', 'default', 'get', 'pod', 'telepresence-1506520541-048964-25778-1327857849-08rns', '-o', 'json'],)...
  18.8 TL | [59] captured.
  19.0 TL | [60] Capturing: (['kubectl', '--context', 'minikube', '--namespace', 'default', 'get', 'pod', 'telepresence-1506520541-048964-25778-1327857849-08rns', '-o', 'json'],)...
  19.2 TL | [60] captured.
  19.4 TL | [61] Capturing: (['kubectl', '--context', 'minikube', '--namespace', 'default', 'get', 'pod', 'telepresence-1506520541-048964-25778-1327857849-08rns', '-o', 'json'],)...
  19.5 TL | [61] captured.
  19.5 TL | [62] Launching: (['kubectl', '--context', 'minikube', '--namespace', 'default', 'logs', '-f', 'telepresence-1506520541-048964-25778-1327857849-08rns', '--container', 'telepresence-1506520541-048964-25778'],)...

```

**Comment by ark3** [comment]
Sorry about the crash; Telepresence should be better about reporting errors rather than crashing. Looks like the AUR package is incorrect. As a workaround, you could grab a copy of [`stamp-telepresence`](https://raw.githubusercontent.com/datawire/telepresence/master/cli/stamp-telepresence) and put it in your PATH somewhere. That should get you going until we fix things the right way.

**Comment by vstepchik** [comment]
Thank you!
After adding `stamp-telepresence` to PATH i've got:
```
Here's the traceback:

Traceback (most recent call last):
  File "/usr/bin/telepresence", line 257, in call_f
    return f(*args, **kwargs)
  File "/usr/bin/telepresence", line 2350, in go
    runner, args
  File "/usr/bin/telepresence", line 1516, in start_proxy
    processes, socks_port, ssh = connect(runner, remote_info, args)
  File "/usr/bin/telepresence", line 1099, in connect
    bufsize=0,
  File "/usr/bin/telepresence", line 404, in popen
    return self.launch_command(track, *args, **kwargs)
  File "/usr/bin/telepresence", line 367, in launch_command
    stderr=self.logfile
  File "/usr/lib/python3.6/subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1333, in _execute_child
    raise child_exception_type(errno_num, err_msg)
NotADirectoryError: [Errno 20] Not a directory
```

**Comment by ark3** [comment]
Sorry, my suggestion was worded poorly. What I meant to say is to place `stamp-telepresence` in a directory that is in your PATH. For example, if `/usr/local/bin` is in your PATH, you can place `stamp-telepresence` there.

Also, you will need to make `stamp-telepresence` executable. Something like `chmod a+x /path/to/stamp-telepresence` should do it.

**Comment by vstepchik** [comment]
Ok that worked, my guess is that telepresence was trying access the file as the root user, and as root the file was not on the path. Thanks!

**Comment by starkers** [comment]
Sorry that was my fault, I updated the arch repo to install 0.67 but I didn't notice the new executable (or test the AUR)

**Comment by ark3** [comment]
With @starkers's fix in place, it looks like Telepresence 0.67 should install correctly on Arch Linux now. @vstepchik does it work for you? If it does, then we can close this bug. I'll continue work on Arch Linux stuff in #135.

**Comment by vstepchik** [comment]
@starkers @ark3 yes, it's fixed, thanks guys!
